index.es.md

```diff
@@ -79,7 +79,7 @@ Por “contenido” web se hace referencia, generalmente, a la información que
 Las pautas WCAG se aplican al contenido dinámico, multimedia, “móvil”, etc. También se pueden aplicar a tecnologías de la información y la comunicación (TIC) no relacionadas con la web, como se explica en [WCAG2ICT](/standards-guidelines/wcag/non-web-ict/).
 
 Información sobre las pautas WCAG:
-- [Resumen de WCAG](  /standards-guidelines/wcag/)
+- [Introducción a las WCAG](  /standards-guidelines/wcag/)
```

Comment:
Would you mind changing this to 

```md
[[Introducción a las WCAG]](/standards-guidelines/wcag/)
```

Especially removing the spaces before the first slash. When you use the double square brackets, the link will automatically use the title of the linked-to site (when a translation exists, not available in the preview). Therefore taking the burden away from you to go back and redo all the links when the title of a document changes or when a document is not translated and you put in a temporary title.

See: https://www.w3.org/wiki/WAI/Website/Anatomy_of_a_Document#Title_Markdown_Links:_.5B.5BTitle_of_the_Page.5D.5D.28.2Fpath.2Fto.2Fpage.2F.29